data/news/platform/ocaml-5.1.alpha1.md

```diff
@@ -0,0 +1,71 @@
+---
+title: OCaml 5.1.0 - First Alpha
+description: First Alpha Release of OCaml 5.1.0
+date: "2023-04-15"
+tags: [ocaml, platform]
+---
+
+Four months after the release of OCaml 5.0.0, the set of new features for the
+future version 5.1.0 of OCaml has been frozen. we are thus happy to announce the
+first alpha release for OCaml 5.1.0 .
+
+This alpha version is here to help fellow hackers join us early in our bug
+hunting and opam ecosystem fixing fun (see below for the installation instructions).
+The progresses on stabilizing the ecosystem are tracked on the [Opam Readiness for 5.1.0 meta-issue](https://github.com/ocaml/opam-repository/issues/23669).
+
+If you find any bugs, please report them on  [OCaml's issue tracker](https://github.com/ocaml/ocaml/issues).
+
+Note that this early alpha version is missing two important fixes for the garbage
+collector and Windows support. Those fixes will be available before the beta.
+The full release is expected to happen in July.
+
+If you are interested by the ongoing list of new features and bug fixes, the
```

Comment:
The changelog tracks the full list of changes, rather than just "updates". 
Maybe

> If you are interested in ongoing list of new features and bug fixes

?